### PR TITLE
fix(CNavigation): Identify offsets for up and right vectors

### PR DESCRIPTION
**Pull request by davidebeatrici**
The class is very similar to `CMatrix` from San Andreas.

https://github.com/Updated-Classic/gta-reversed-modern/blob/2e063063bed74690a4b6e9393f26f2b94ee4c1cb/source/game_sa/Core/Matrix.h

### CODE REVIEW

**Comment by davidebeatrici** [comment]
Should I update `gtav.rcnet`? If yes, how?

**Comment by Yimura** [comment]
Thank you for your contribution first of all!

The `gtav.rcnet` is what I use to map these structures to in-game memory, it also allows me to see some RTTI from classes/structs.

It's mostly just offered as a step-in for new people who haven't used [ReClass.NET](https://github.com/ReClassNET/ReClass.NET) before.
The `.rcnet` contains more information than is currently in this repo and is for my own personal research, the file can't really be used by multiple people at the same time.

In short I'll update the file the next time I work on it.

**Comment by davidebeatrici** [comment]
Okay, thanks!

**Comment by Yimura** [comment]
I just went ahead and fixed the naming scheme.
